graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/ValidationErrorInstrumentation.kt

```diff
@@ -0,0 +1,34 @@
+package com.netflix.graphql.dgs.internal
+
+import com.netflix.graphql.dgs.exceptions.DgsException
+import com.netflix.graphql.types.errors.ErrorDetail
+import com.netflix.graphql.types.errors.ErrorType
+import com.netflix.graphql.types.errors.TypedGraphQLError
+import graphql.ExecutionResult
+import graphql.GraphQLError
+import graphql.execution.instrumentation.InstrumentationState
+import graphql.execution.instrumentation.SimplePerformantInstrumentation
+import graphql.execution.instrumentation.parameters.InstrumentationExecutionParameters
+import graphql.validation.ValidationError
+import java.util.concurrent.CompletableFuture
+
+class ValidationErrorInstrumentation : SimplePerformantInstrumentation() {
+    override fun instrumentExecutionResult(executionResult: ExecutionResult?, parameters: InstrumentationExecutionParameters?, state: InstrumentationState?): CompletableFuture<ExecutionResult> {
```

Comment:
nit: you might be able to get rid of the null checks as the parameters are likely non nullable.